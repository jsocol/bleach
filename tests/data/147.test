<div id="76"><?xml-stylesheet type="text/css"?><!DOCTYPE x SYSTEM "test.dtd"><x>&x;</x>//["'`-->]]>]</div>
