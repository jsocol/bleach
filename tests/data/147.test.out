<div id="76">&amp;x;//["'`--&gt;]]&gt;]</div>
